shared/protocol/src/test/java/io/pravega/shared/protocol/netty/WireCommandsTest.java

```diff
@@ -621,6 +621,15 @@ public void testKeepAlive() throws IOException {
         testCommand(new WireCommands.KeepAlive());
     }
 
+    @Test(expected = IOException.class)
+    public void testFlush() throws IOException {
```

Comment:
All this test is doing is validating that the new command is constructed correctly. Even though such a test should exist, I expect tests that validate the behavior as well. Do such tests exist already?

Reply:
we already wave testCommand to test the write-fields (serialize to wire) and readFrom - deserialize ; but this new command is should not write anything to wire. hence , a new test to just check its constructed properly and ensures that writeFields (serialize) will throw an IO error.